app.js

```diff
@@ -1,9 +1,97 @@
 // in server.js
 const express = require('express');
 const app = express();
+const cors = require('cors')
+const pgp = require('pg-promise')
+const bodyParser = require('body-parser')
+const questions = require('./src/database/queries/questions.js')
 
-app.use(express.static(__dirname + '/public/dist/'));
-app.use(express.static(__dirname + '/src/browser/'));
+app.use( bodyParser.json() )
+app.use( bodyParser.urlencoded( {extended: false}) )
+app.use( express.static(__dirname + '/public/dist/') );
+app.use( express.static(__dirname + '/src/browser/') );
+app.use( ( cors() )
+
+app.get('/users/:name', (request, response) => {
```

Comment:
please create a new file called `src/server/routes/users.js`, and put the user routes in there